indico/modules/events/registration/__init__.py

```diff
@@ -106,7 +107,8 @@ def _visible_registration(event):
                 .has_rows())
 
     def _visible_participant_list(event):
-        return event.has_feature('registration')
+        override = values_from_signal(signals.event.visible_participant_list.send(sender, event=event))
+        return event.has_feature('registration') if not override else override.pop()
```

Comment:
Hello @ThiefMaster , In this case where `override.pop()` was used, this was considering that only one plugin is subscribed to the signal. But in the case that there are multiple plugins subscribed, would it then be incorrect to only consider the first value ?